.travis.yml

```diff
@@ -7,32 +7,41 @@ matrix:
     - os: windows
       env: BUILD_CONFIGURATION=Release
     - os: linux
-      dist: trusty
-      group: travis_lts
-      sudo: required
-      env: BUILD_CONFIGURATION=Debug
+      compiler: gcc
+      addons: &gcc9
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['g++-9', 'gcc-9']
+      env: GCCVER='9' CXX_COMPILER=g++-${GCCVER} CXX=${CXX_COMPILER} C_COMPILER=gcc-${GCCVER} CC=${C_COMPILER} AR=gcc-ar-${GCCVER} RANLIB=gcc-ranlib-${GCCVER} NM=gcc-nm-${GCCVER} BUILD_CONFIGURATION=Release
```

Comment:
Please, move `BUILD_CONFIGURATION=Release` at the very beginning, just to emphasize which kind of build it is I think it is convenient.

Reply:
ok